oidc_example/op1/oc_server.py

```diff
@@ -1,52 +1,44 @@
 #!/usr/bin/env python
 import json
+import logging
 import os
 import re
 import sys
 import traceback
-from exceptions import AttributeError
-from exceptions import Exception
-from exceptions import IndexError
-from exceptions import KeyboardInterrupt
-from exceptions import KeyError
-from exceptions import OSError
 from logging.handlers import BufferingHandler
-from urlparse import parse_qs
 
+from exceptions import AttributeError, Exception, IndexError, KeyboardInterrupt, KeyError, OSError
 from mako.lookup import TemplateLookup
```

Comment:
This looks a little surprising from a isort/pep8 styleguide perspective. 
mako.lookup should be third party and not lined up with the stdlib stuff.

But if ruff thinks its the way, I'm fine with it.